plugins/deviantart.py

```diff
@@ -107,8 +107,10 @@ def import_submission(self, submission: praw.objects.Submission) -> dict:
             data = {'author': response['author_name'],
                     'source': submission.url,
                     'importer_display':
-                        {'header': 'Mirrored deviantArt image by the author "{}":\n\n'.format(
-                            response['author_name'])}}
+                        {'header': 'Mirrored [deviantArt image]({post_url}) by the [author "{author_name}"]({author_url}):\n\n'.format(
+                            post_url=submission.url, 
+                            author_name=response['author_name'], 
+                            author_url=response['author.url'])}}
```

Comment:
Where does `author.url` come from? Why not `author_url` for consistency with above?